test/integration/service_manual_topic_test.rb

```diff
@@ -40,6 +40,13 @@ class ServiceManualTopicTest < ActionDispatch::IntegrationTest
     end
   end
 
+  test "it includes a link to subscribe for email alerts" do
+    setup_and_visit_example('service_manual_topic', 'service_manual_topic')
+
+    assert page.has_link?("email",
+      href: "http://www.dev.gov.uk/service-manual/test-expanded-topic/email-signup")
```

Comment:
A minor thing. Matching just the path will make this slightly more robust. You can use a regex.